chat.go

```diff
@@ -80,6 +86,7 @@ func GetChatRoom(name string, password string, joinable, fixed bool) (room *Chat
 	return room
 }
 
+// run is the main loop that runs and handles events for a ChatRoom.
 func (cr *ChatRoom) run() {
 	defer func() {
```

Comment:
You could, although if memory serves it's probably a bit redundant to have all of these functions public as `ChatRoom`s are essentially an isolated environment aside from the channels used to add and remove users.